src/mmrelay/migrate.py

```
@@ -0,0 +1,1282 @@
+            except (OSError, IOError) as e:
+                logger.warning("Failed to backup database: %s", e)
+
+        try:
+            if move:
+                logger.info("Moving database file: %s", db_path)
+                shutil.move(str(db_path), str(dest))
+            else:
+                logger.info("Copying database file: %s", db_path)
+                shutil.copy2(str(db_path), str(dest))
+        except (OSError, IOError) as e:
+            logger.error("Failed to migrate database file %s: %s", db_path, e)
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)